core/src/utils/test/playwright/generator.ts

```diff
@@ -28,6 +31,11 @@ interface TestUtilities {
 interface TestConfigOption {
   modes?: Mode[];
   directions?: Direction[];
+  themeModes?: ThemeMode[];
+  /**
+   * The individual themes (iOS, Material Design and Ionic) to test
+   * against. If unspecified, defaults iOS and Material Design
```

Comment:
This would create a significant number of new screenshots. This behavior only exists for backwards compatibility. 

Ideally we do not test against any specific theme by default and the developer specifies the theme they would like to test against.

Reply:
Sounds good to me